src/pages/Detail/Modal/ModalApply.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const ModalApply = props => {
+  const [addFile, setAddFile] = useState('');
+  const [uploadValue, setValue] = useState('');
+
+  const onInputChange = e => {
+    setAddFile([...addFile, ...e.target.files]);
+    setValue(e.target.value);
+  };
+
+  const onSubmit = e => {
+    e.preventDefault();
+    const formData = new FormData();
+
+    for (let i = 0; i < addFile.length; i++) {
+      formData.append('file', addFile[i]);
+    }
```

Comment:
```suggestion
    addFile.forEach(file => formData.append("file",file));
```
forEach 메소드를 이용해서도 할 수 있습니다!
기본적이고 절차적인 for 문보다는 forEach, map, reduce 등의 선언적인 메소드들을 잘 활용하시는게 좋습니다!